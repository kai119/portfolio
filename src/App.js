import * as React from "react";
import { NextUIProvider } from "@nextui-org/system";

function App() {
  return (
    <NextUIProvider>
      <div />
    </NextUIProvider>
  );
}

export default App;
