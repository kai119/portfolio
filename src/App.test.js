import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock("./pages/HomePage", () => {
  const HomePage = () => <div />;
  return HomePage;
});

it("renders the app with default props", () => {
  const { container } = render(<App />);
  const { firstChild } = container;
  expect(firstChild).toMatchSnapshot();
});
