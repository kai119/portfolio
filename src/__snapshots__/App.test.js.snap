// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the app with default props 1`] = `<div />`;
